Yoast/Tests/Commenting/CodeCoverageIgnoreDeprecatedUnitTest.inc

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class SomeThing {
+
+	/**
+	 * Active (non-deprecated) function.
+	 */
+	function activeFunction() {}
+
+	/**
+	 * @deprecated x.x
+	 * @codeCoverageIgnore
+	 */
+	public /* don't get confused by this */ function correctDocblock() {
+		// Code
+    } // end function
```

Comment:
please notice this line uses spaces for indentation, it should use a tab character

Reply:
@afercia Hi Andrea, FYI: that is the unit test file for the sniff and the sniff should work style-independently, i.e. detect the issue it is targeting no matter what other other CS issues are present.

So, while you are correct, in this case, it really doesn't matter and could be considered a feature.

Reply:
@jrfnl hi! Thanks for the explanation. Hm then maybe consider an inline comment to explain it? Otherwise I'm afraid some pedant guy like me might inadvertently change them to tab, thinking to do the right thing 😄

Reply:
Sorry I'm afraid I don't have much time, pretty busy on a new ESLint config for _all_ the projects 😬

Reply:
Very valuable as well. I'll make an issue for the Contributing file.

Reply:
See #97